### PR TITLE
templates: Edit welcome.md (Addresses #1732)

### PR DESCRIPTION
**Pull request by RaoOfPhysics**
Happy to accept any comments before we merge this.

Signed-off-by: Achintya Rao <achintya.rao@cern.ch>

### CODE REVIEW

**Comment by RaoOfPhysics** [comment]
Looks like @annatrz also edited the `welcome.json` file. Oops! I managed to successfully unstage my changes and rebase against master. New success for me! :D Can be reviewed now, I guess. :)

**Comment by ioannistsanaktsidis** [comment]
@RaoOfPhysics please remove the ```addresses``` part from the title and add it as a bullet.
Sth like
```
templates: Edit welcome.md 

 * Addresses #1732

Signed-off-by: ....
```

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
+CERN Open Data provides content for both education and research. We aim to support high-school students and teachers as well as university students and professors. For this purpose, the new design of CERN Open Data is article-driven. Detailed [guides](/search?type=Documentation&subtype=Guide) introduce you to physics analyses and make use of [datasets available from this portal](/search?type=Dataset). Event displays allow to visualise collision data and examine what happens when particles interact in high-energy collisions.
+
+Even though enthusiastic members of the wider public as well educators in particular have been on our mind when we redesigned the portal, we did not forget about researchers. It’s quite the opposite actually: our new search features and updated facets allow for an even more efficient and effective access to relevant [datasets](/search?type=Dataset). We are proud that data released on CERN Open Data led to the [first publication](https://arxiv.org/abs/1704.05842) of a research paper using LHC data by researchers not belonging to any LHC collaboration. This is truly open science!
```
Comment (suenjedt) [inline cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: Can we re-shuffle the phrase a little bit? To me the sentence is confusing. Something like: We are proud that previous data releases on CERN Open Data  have already yielded new results: first publications of research using LHC data written by researchers not belonging to any LHC collaboration came out in 2017. [probably too long, sorry]

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 
-Even though the general public and educators have been on our mind when we redesigned the portal, we did not forget about researchers. It’s quite the opposite actually. Our new search features and updated facets allow for an even more efficient and effective access to relevant [datasets](/search?type=Dataset). We are proud that data released on CERN Open Data led to the [first publication](https://arxiv.org/abs/1704.05842) of a research paper outside of any LHC collaboration.
+The success of CERN Open Data is made possible through working closely with various research collaborations affiliated with CERN. We provide access to data and resources from the four big LHC collaborations: [ALICE](/about/alice), [ATLAS](/about/atlas), [CMS](/about/cms) and [LHCb](/about/lhcb).
```
Comment (suenjedt) [inline cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: ...affiliated with CERN and external partners.

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md [hunk 3]
```diff
@@ -1,14 +1,12 @@
...
 
-The success of CERN Open Data is made possible through the close connections to various collaborating experiments. So far, we provided access to data and resources from the LHC collaborations [ALICE](/about/alice), [ATLAS](/about/atlas), [CMS](/about/cms) and [LHCb](/about/lhcb). We are proud to announce that together with the release of the updated portal, we are able to welcome the [OPERA](/about/opera) experiment. 
- 
-We hope that you like our updated portal and appreciate any feedback and comments from your side. Please don’t hesitate to [contact](mailto:opendata-support@cern.ch) us.
+We hope that you like our updated portal and appreciate any feedback and comments from your side. Please don’t hesitate to [contact](mailto:opendata-support@cern.ch) us. You are also welcome to submit issues to [our GitHub repository](https://github.com/cernopendata/opendata.cern.ch/), where development of this portal takes place.
```
Comment (suenjedt) [inline cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: takes place in the open.

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 1 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: Yes, I found the sentence clunky as well when I wrote it, but I pushed this commit in a hurry. Will update. Thanks!

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 2 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: @suenjedt: Do you mean "working closely… with external partners"?

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 1 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: How about (please read the whole paragraph):

> Even though enthusiastic members of the wider public as well educators in particular have been on our mind when we redesigned the portal, we have made sure that our portal caters to researchers as well. Our new search features and updated facets allow for an even more efficient and effective access to relevant [datasets](/search?type=Dataset). We are proud that our previous data releases have already yielded new results: 2017 saw the publication of the [first scientific paper](https://arxiv.org/abs/1704.05842) using LHC data that was authored by external researchers, that is ones not belonging to an LHC collaboration.

Or…

> […] We are proud that our previous data releases have already yielded new results: 2017 saw the publication of the [first scientific paper](https://arxiv.org/abs/1704.05842) using LHC data that was authored by researchers not belonging to an LHC collaboration.

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 2 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: How about:

> The success of CERN Open Data is made possible through working closely with various CERN-affiliated research collaborations and with external partners.

(Not sure which of the two interpretations you meant.)

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 3 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: Done.

**Comment by RaoOfPhysics** [comment]
@suenjedt: I've left some comments to your review above. If you're happy with them, I'll commit and push my changes.

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 1 above)
Comment (suenjedt) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: I like the first one. But re-reading this, I realize that I would like citizen science/educators/researchers on the same level. 
Perhaps like "While we had citizen scientists, the wider public, as well educators in mind when we redesigned the portal, we have of course also made sure that our portal caters to researchers interested in these unique materials.
Feel free to change of course!

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 2 above)
Comment (suenjedt) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: Yes, you nailed it!

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md (hunk 1 above)
Comment (RaoOfPhysics) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: I don't necessarily want to draw a distinction between "citizen scientists" and "academic researchers". But I like your overall restructuring.

> While we had educators, students and the wider public in mind when we redesigned the portal, we have of course also made sure that we cater to the broader research community that may be interested in these unique materials.

?

**Comment by RaoOfPhysics** [comment]
@suenjedt: I've updated the text. @ioannistsanaktsidis: I've amended the commit message.

**Comment by ioannistsanaktsidis** [comment]
@RaoOfPhysics  please rebase against latest master and I will merge.

**Comment by RaoOfPhysics** [comment]
@ioannistsanaktsidis: Done.

**Pull request merged**